doc/source/pbm-commands.rst

```diff
@@ -162,6 +162,40 @@ The command accepts the following flags:
        - ``%Y-%M-%D`` (e.g. 2020-04-20)
    * - ``--force``
      - Forcibly deletes backups without asking for user's confirmation   
+
+.. _delete-pitr:
+
+.. rubric:: pbm delete-pitr
+
+Deletes :term:`oplog slices <Oplog slice>` produced for :ref:`pitr`. 
+
+The command has the following syntax:
+
+.. code-block:: bash
+
+   $ pbm delete-pitr [<flags>] 
+
+The command accepts the following flags:
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Flag
+     - Description
+   * - ``-a``, ``--all``
+     - Deletes all oplog slices
+   * - ``--older-than=TIMESTAMP``
+     - Deletes oplog slices older than date / time specified in the format:
+     
+       - ``%Y-%M-%DT%H:%M:%S`` (e.g. 2020-04-20T13:13:20) or 
+       - ``%Y-%M-%D`` (e.g. 2020-04-20)
+    
+       Note that |PBM| doesn't delete the oplog slices that follow the most recent backup. This is done to ensure point in time recovery from that backup snapshot. For example, if the snapshot is ``2021-07-20T07:05:23Z [complete: 2021-07-21T07:05:44]`` and you specify the timestamp ``2021-07-20T07:05:44``, |PBM| deletes only slices that were made before ``2021-07-20T07:05:23Z`` 
+   * - ``--force``
+     - Forcibly deletes oplog slices without asking a user's confirmation
+   * - ``-o``, ``--out=json``
+     - Shows the output as either the plain text (default) or a JSON object. Supported values: ``json``, ``json-pretty``.
```

Comment:
It's also worth mentioning that if `--older-than` set, it will be rounded down to the last write of the closest preceding backup in order not to make gaps. So usually it gonna leave some extra chunks. E.g. if `older-than = 13` and the last write of the closest preceding backup is `10` it will leave `11` and `12` chunks as well since `13` won't be restorable without `11` and `12` (contiguous timeline from the backup).

Reply:
Added